packages/recommend-core/src/personalization/getStrategy.ts

```diff
@@ -0,0 +1,34 @@
+type GetStrategy = {
+  appId: string;
+  apiKey: string;
+  region: string;
+};
+
+type StrategyResponse = {
+  facetsScoring: Array<{
+    facetName: string;
+    score: number;
+  }>;
+  personalizationImpact: number;
+};
```

Comment:
Just making sure that you don't need the [`eventsScoring` param](https://www.algolia.com/doc/rest-api/personalization/#method-param-eventsscoring)?